tests/test_dimension.py

```diff
@@ -1072,11 +1072,12 @@ def test_no_fusion_simple(self):
                 Eq(g, f + 1, implicit_dims=[ctime])]
 
         op = Operator(eqns)
-        exprs = FindNodes(Expression).visit(op._func_table['bf0'].root)
+        _, bns = get_blocked_nests(op, {'x0_blk0', 'x1_blk0'})
```

Comment:
imho all these calls to `assert_blocking` want blank lines around

Reply:
So, not as before right?
i.e. we add two empty lines ?